src/activations.rs

```diff
@@ -314,6 +395,27 @@ mod tests {
         assert_eq!(input, expected);
     }
 
+    #[test]
+    fn test_switsh_1d(){
+        let data: Array1<f32> = Array1::from_shape_vec(4, vec![-2.0, 2.0, 4.0, -7.0]).unwrap();
+
+        let result: Array1<f32> = Activation::Swish.activation(&data);
+        let expected: Array1<f32> = array![-0.23840584,  1.761594,  3.92805516, -0.006377358];
+
+        assert_eq!(expected,result);
+    }
+
+    #[test]
+    fn test_switsh_2d(){
+        let data:Array2<f32> = Array2::from_shape_vec((2,2), vec![-2.0, 2.0, 4.0, -7.0]).unwrap();
+        
+        let result:Array2<f32> = Activation::Swish.activation(&data);
+        let expected:Array2<f32> = array![[-0.23840584,  1.761594],
+                                        [3.92805516, -0.006377358]];
```

Comment:
If you please apply `rustfmt` so we have all the formatting consistent.
```suggestion
        let data: Array2<f32> = Array2::from_shape_vec((2,2), vec![-2.0, 2.0, 4.0, -7.0]).unwrap();
        
        let result: Array2<f32> = Activation::Swish.activation(&data);
        let expected: Array2<f32> = array![[-0.23840584,  1.761594],
                                        [3.92805516, -0.006377358]];
```

Reply:
I've formatted and pushed the changes